msg/RadarPoint.msg

```diff
@@ -0,0 +1,10 @@
+# All variables below are relative to the radar's frame of reference.
```

Comment:
Just a thought: would it make sense to name this message `RadarReturn` instead of `RadarPoint`?
Feels a little more intuitive (just my opinion) and aligns more with radar terminology.